src/database/standalone-sqlite.ts

```
@@ -4402,6 +4451,14 @@ if (!isMainThread) {
           worker.saveDataContentAttributes(args[0]);
           parentPort?.postMessage(null);
           break;
+        case 'getManifestResolution':
+          const manifestResolution = worker.getManifestResolution(args[0]);
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)